### DISCUSSION TITLE
能否支持调整client请求时间间隔的参数

### DISCUSSION THREAD

**Discussion by OnlyFor** [zdz/ServerStatus-Rust#123] [Ideas]
我看了一下代码，目前是写死client每隔1s发送请求的，能否支持自定义时间间隔？
比如我机器很多，然后如果流量很贵或者需求比较低频的话，可能希望调整时间间隔为3s或者5s请求一下就够了

希望能支持一下，感谢

**Comment by zdz** [comment]
现在使用 pb 协议上报，单次请求大小 600 bytes 左右，每月1.5G 左右的 流量，其实还好吧，现在的 vps 流量都是按 T 来算
先记下，下个版本加个参数支持一下

**Reply by OnlyFor** [reply]
感谢大佬，要是机器多的话总量还是有点的，还有就是亚洲机器普遍流量贵。。。

**Comment by zdz** [comment]
[v1.6.2](https://github.com/zdz/ServerStatus-Rust/releases/tag/v1.6.2) 已支持

**Reply by OnlyFor** [reply]
感谢
